src/Lykke.Service.Decred.Api.Repository/Output.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Collections.Generic;
+using Decred.BlockExplorer;
+
+namespace Lykke.Service.Decred.Api.Repository
+{
+    public class Output
+    {
+        public string Hash { get; set; }
+
+        private sealed class HashOutputIndexEqualityComparer : IEqualityComparer<Output>
+        {
+            public bool Equals(Output x, Output y)
+            {
+                if (ReferenceEquals(x, y)) return true;
+                if (ReferenceEquals(x, null)) return false;
+                if (ReferenceEquals(y, null)) return false;
+                if (x.GetType() != y.GetType()) return false;
+                return string.Equals(x.Hash, y.Hash) && x.OutputIndex == y.OutputIndex;
+            }
+
+            public int GetHashCode(Output obj)
```

Comment:
Calculating hash using muttable properties is not safe. Might this class be immutable?